File: leetcode/1easy/이경호/3127.cpp
class Solution {
  public:
      bool canMakeSquare(vector<vector<char>>& grid) {
          for(int i = 0; i < 2; i++) {
              for(int j = 0; j < 2; j++) {
                  int b = 0;
                  int w = 0;
                  for(int k = 0; k < 2; k++) {
                      for(int r = 0; r < 2; r++) {
                          if(grid[i + k][j + r] == 'B') {
                              b++;
                          }
                          else {
                              w++;
                          }
                      }
                  }
                  if(b >= 3 || w >= 3) {
                      return true;
                  }
              }
          }
          return false;
      }
  };

File: leetcode/1easy/김민진/3127.py
class Leetcode.이재훈.Solution:
    def canMakeSquare(self, grid: List[List[str]]) -> bool:
        for i in range(2):
            for j in range(2):
                n_black = grid[i][j:j+2].count("B") + grid[i+1][j:j+2].count("B")
                if n_black != 2:
                    return True
        else:
            return False
        